src/runtime/Makefile

```diff
@@ -112,6 +112,7 @@ KERNELDIR := $(PKGDATADIR)
 
 IMAGEPATH := $(PKGDATADIR)/$(IMAGENAME)
 INITRDPATH := $(PKGDATADIR)/$(INITRDNAME)
+DEFROOTFSTYPE := \"ext4\"
```

Comment:
I'd suggest to switch to erofs by default if possible. I think the benefit is enough

Reply:
we can show benefits explicitly here, but it can be done in a separated PR

Reply:
I switch EROFS as default rootfs filesystem.

I think might be better to show the benefits of EROFS in image_builder.sh. I will add it when the documentation is complete later.